lib/elastic_apm/span.rb

```diff
@@ -75,6 +70,15 @@ def running?
       started? && !stopped?
     end
 
+    def stacktrace
+      return @stacktrace if defined? @stacktrace
```

Comment:
We can skip `defined?`?

Reply:
I think so. This was meant to memoize `stacktrace`